core/src/main/java/hudson/model/Run.java

```diff
@@ -2422,7 +2422,7 @@ public String getEntryID(Run e) {
 
     private static class DefaultFeedAdapter implements FeedAdapter<Run> {
         public String getEntryTitle(Run entry) {
-            return entry.getDisplayName()+" ("+entry.getBuildStatusSummary().message+")";
+            return entry.getFullDisplayName();
```

Comment:
your first commit isnt right becouse if default build name changed we will lost project build name in rss feed